frontend/udp/frontend.go

```diff
@@ -286,16 +286,17 @@ func (t *Frontend) handleRequest(r Request, w ResponseWriter) (actionName string
 		af = new(bittorrent.AddressFamily)
 		*af = req.IP.AddressFamily
 
+		var ctx context.Context
```

Comment:
I did it to match resp; I can change the assignment to set the value on declaration.